BitGo/wallet-recovery-wizard
# Add BCHA to Unsigned Sweep and Non BitGo Recoveries

This PR adds BCHA recovery to unsigned sweep and non-bitgo recoveries (for mainnet only) 

Testing Instruction:

1. Create a Prod BCH Wallet and fund it
2. Input the wallet keycard information into the WRW Unsigned sweep (cold) non-bitgo (hot)
3. Download the transaction json
4. Broadcast the txhex to the bitcoin-cash network (equivalent to broadcasting to the bitcoin-abc network since pre-fork)
```
 curl -X GET "https://rest.bitcoin.com/v2/rawtransactions/sendRawTransaction/{TXHEX}" -H "accept: */*"
```
5. Look up your recovery destination address on the bitcoin-abc, to make sure that it has received the recovered amounts 
`https://api.blockchair.com/bitcoin-abc/dashboards/address/ADDRESS`

--------- 
How I tested this:

A cold wallet with my own user and backup keys [Transaction successfully broadcasted]
A Warm wallet with my own user key and Coincover’s backup key [Transaction created/downloaded successfully' waiting for Coincover to cosign]
A hot wallet with Coincover As KRS [Transaction created/downloaded successfully' waiting for Coincover to cosign]
A hot wallet with Browser created backup key [Transaction successfully broadcasted]
